Documentation/ApiOverview/Authentication/Index.rst

```diff
@@ -84,6 +84,26 @@ by a login request. In the FE, this happens when a form field
 called "logintype" is submitted with value "login". The same
 happens for the BE, but with a form field called "login_status".
 
+..  versionchanged:: 12.0
+    `JSON Web Tokens (JWT) <https://jwt.io/>`__ are used to transport user
+    session identifiers in `be_typo_user` and `fe_typo_user` cookies.
+
+Using JWT's `HS256` (HMAC signed based on SHA256) allows to determine whether a
+session cookie is valid before comparing with server-side stored session data.
+This enhances the overall performance a bit, since sessions cookies would be
+checked for every request to TYPO3's backend and frontend.
+
+The session cookies can be pre-validated without querying the database, which
+can filter invalid requests and might reduce the enhances the overall
+performance a bit.
+
+As a consequence session tokens are not sent "as is", but are wrapped in a
```

Comment:
```suggestion
As a consequence session tokens are not sent "as are", but are wrapped in a
```

Reply:
Nope, "as is" should be right: https://dict.leo.org/englisch-deutsch/as%20is

Reply:
It sounds wrong, I asked Tom as native speaker

Reply:
Ok, Tom agrees with you